.gitignore

```diff
@@ -24,4 +24,5 @@ package-lock.json
 /dist
 
 /lnd_data
+.vscode/
```

Comment:
Please use a global gitignore for your machine, as everyone uses a different editor.